dashboard/templatetags/cards.py

```diff
@@ -235,22 +238,40 @@ def _feeding_statistics(child):
     :param child: an instance of the Child model.
     :returns: a dictionary of statistics.
     """
+    feedings = [
+            {
+                'start': timezone.now()-timezone.timedelta(days=3), 
+                'title': _('Feeding frequency')+ " 3 "+_("days")
+            }, 
+            {
+                'start': timezone.now()-timezone.timedelta(weeks=2), 
+                'title': _('Feeding frequency') + " 2 "+_("weeks")
+            },
+            {
+                'start': timezone.make_aware(datetime.combine(date.min, time(0,0)) + timezone.timedelta(days=1) ),
+                'title': _('Feeding frequency')
+            }
+        ]
```

Comment:
There is no need to concat the translated strings here. "Feeding frequency" is already a translation string, but "days" and "weeks" are not. Anyway, I'd prefer to have these strings as "Feeding frequency (past 3 days)" and "Feeding frequency (past 2 weeks)".